src/apps/js_v8/kv_module_loader.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+
+#include "kv_module_loader.h"
+
+#include "ccf/tx.h"
+#include "node/entities.h"
+#include "node/modules.h"
+
+namespace ccf
+{
+  std::optional<std::string> v8_kv_module_load_callback(
+    const std::string& module_name, void* opaque)
+  {
+    auto tx = (kv::Tx*)opaque;
```

Comment:
I know we're not particularly good at this in the codebase, but I think we ought to try to stick to C++ casts as much as possible.